Panic when creating new cluster with cloudWatch.clusterLogging

**What happened?**
Creating a new cluster from config file resulted in a `panic: runtime error: invalid memory address or nil pointer dereference`

**What you expected to happen?**
No panic, and a cluster with control plane logging enabled.

**How to reproduce it?**
`$ eksctl create cluster -f examples/01-simple-cluster.yaml`

**Anything else we need to know?**
MacOS 10.14.6
brewed eksctl
AWS MFA, assumed role

**Versions**
```
$ eksctl version
[ℹ]  version.Info{BuiltAt:"", GitCommit:"", GitTag:"0.3.0"}

$ kubectl version
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.2", GitCommit:"f6278300bebbb750328ac16ee6dd3aa7d3549568", GitTreeState:"clean", BuildDate:"2019-08-05T16:54:35Z", GoVersion:"go1.12.7", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.8-eks-a977ba", GitCommit:"a977bab148535ec195f12edc8720913c7b943f9c", GitTreeState:"clean", BuildDate:"2019-07-29T20:47:04Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Logs**
```
$ eksctl create cluster -f examples/01-simple-cluster.yaml -v 4
2019-08-07T12:28:50+02:00 [ℹ]  using region eu-north-1
2019-08-07T12:28:51+02:00 [▶]  role ARN for the current session is "arn:aws:sts::<redacted>:assumed-role/<redacted>/<redacted>"
2019-08-07T12:28:51+02:00 [▶]  determining availability zones
2019-08-07T12:28:51+02:00 [ℹ]  setting availability zones to [eu-north-1c eu-north-1b eu-north-1a]
2019-08-07T12:28:51+02:00 [▶]  VPC CIDR (192.168.0.0/16) was divided into 8 subnets [192.168.0.0/19 192.168.32.0/19 192.168.64.0/19 192.168.96.0/19 192.168.128.0/19 192.168.160.0/19 192.168.192.0/19 192.168.224.0/19]
2019-08-07T12:28:51+02:00 [ℹ]  subnets for eu-north-1c - public:192.168.0.0/19 private:192.168.96.0/19
2019-08-07T12:28:51+02:00 [ℹ]  subnets for eu-north-1b - public:192.168.32.0/19 private:192.168.128.0/19
2019-08-07T12:28:51+02:00 [ℹ]  subnets for eu-north-1a - public:192.168.64.0/19 private:192.168.160.0/19
2019-08-07T12:28:51+02:00 [▶]  resolving AMI using StaticGPUResolver for region eu-north-1, instanceType m5.large and imageFamily AmazonLinux2
2019-08-07T12:28:51+02:00 [▶]  can't resolve AMI using StaticGPUResolver as instance type m5.large is non-GPU
2019-08-07T12:28:51+02:00 [▶]  resolving AMI using StaticDefaultResolver for region eu-north-1, version 1.13, instanceType m5.large and imageFamily AmazonLinux2
2019-08-07T12:28:51+02:00 [ℹ]  nodegroup "ng-1" will use "ami-0fd05922165907b85" [AmazonLinux2/1.13]
2019-08-07T12:28:51+02:00 [ℹ]  using Kubernetes version 1.13
2019-08-07T12:28:51+02:00 [ℹ]  creating EKS cluster "cluster-1" in "eu-north-1" region
2019-08-07T12:28:51+02:00 [▶]  cfg.json = \
{
    "kind": "ClusterConfig",
    "apiVersion": "eksctl.io/v1alpha5",
    "metadata": {
        "name": "cluster-1",
        "region": "eu-north-1",
        "version": "1.13"
    },
    "iam": {},
    "vpc": {
        "cidr": "192.168.0.0/16",
        "subnets": {
            "private": {
                "eu-north-1a": {
                    "cidr": "192.168.160.0/19"
                },
                "eu-north-1b": {
                    "cidr": "192.168.128.0/19"
                },
                "eu-north-1c": {
                    "cidr": "192.168.96.0/19"
                }
            },
            "public": {
                "eu-north-1a": {
                    "cidr": "192.168.64.0/19"
                },
                "eu-north-1b": {
                    "cidr": "192.168.32.0/19"
                },
                "eu-north-1c": {
                    "cidr": "192.168.0.0/19"
                }
            }
        },
        "autoAllocateIPv6": false,
        "nat": {
            "gateway": "Single"
        }
    },
    "nodeGroups": [
        {
            "name": "ng-1",
            "ami": "ami-0fd05922165907b85",
            "amiFamily": "AmazonLinux2",
            "instanceType": "m5.large",
            "privateNetworking": false,
            "securityGroups": {
                "withShared": true,
                "withLocal": true
            },
            "desiredCapacity": 1,
            "volumeSize": null,
            "volumeType": "gp2",
            "volumeName": "/dev/xvda",
            "volumeEncrypted": false,
            "volumeIOPS": null,
            "labels": {
                "alpha.eksctl.io/cluster-name": "cluster-1",
                "alpha.eksctl.io/nodegroup-name": "ng-1"
            },
            "ssh": {
                "allow": false
            },
            "iam": {
                "withAddonPolicies": {
                    "imageBuilder": false,
                    "autoScaler": false,
                    "externalDNS": false,
                    "certManager": false,
                    "appMesh": null,
                    "ebs": false,
                    "fsx": false,
                    "efs": false,
                    "albIngress": false,
                    "xRay": false,
                    "cloudWatch": false
                }
            }
        }
    ],
    "availabilityZones": [
        "eu-north-1c",
        "eu-north-1b",
        "eu-north-1a"
    ],
    "cloudWatch": {
        "clusterLogging": {
            "enableTypes": [
                "audit",
                "authenticator",
                "controllerManager"
            ]
        }
    }
}

2019-08-07T12:28:51+02:00 [ℹ]  1 nodegroup (ng-1) was included
2019-08-07T12:28:51+02:00 [ℹ]  will create a CloudFormation stack for cluster itself and 1 nodegroup stack(s)
2019-08-07T12:28:51+02:00 [ℹ]  if you encounter any issues, check CloudFormation console or try 'eksctl utils describe-stacks --region=eu-north-1 --name=cluster-1'
2019-08-07T12:28:51+02:00 [ℹ]  3 sequential tasks: { create cluster control plane "cluster-1", create nodegroup "ng-1", 1 task: { update CloudWatch logging configuration } }
2019-08-07T12:28:51+02:00 [▶]  started task: create cluster control plane "cluster-1"
2019-08-07T12:28:51+02:00 [ℹ]  building cluster stack "eksctl-cluster-1-cluster"
2019-08-07T12:28:51+02:00 [▶]  CreateStackInput = {
  Capabilities: ["CAPABILITY_IAM"],
  StackName: "eksctl-cluster-1-cluster",
  Tags: [{
      Key: "alpha.eksctl.io/cluster-name",
      Value: "cluster-1"
    },{
      Key: "eksctl.cluster.k8s.io/v1alpha1/cluster-name",
      Value: "cluster-1"
    }],
  TemplateBody: "{\"AWSTemplateFormatVersion\":\"2010-09-09\",\"Description\":\"EKS cluster (dedicated VPC: true, dedicated IAM: true) [created and managed by eksctl]\",\"Resources\":{\"ClusterSharedNodeSecurityGroup\":{\"Type\":\"AWS::EC2::SecurityGroup\",\"Properties\":{\"GroupDescription\":\"Communication between all nodes in the cluster\",\"Tags\":[{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/ClusterSharedNodeSecurityGroup\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"ControlPlane\":{\"Type\":\"AWS::EKS::Cluster\",\"Properties\":{\"Name\":\"cluster-1\",\"ResourcesVpcConfig\":{\"SecurityGroupIds\":[{\"Ref\":\"ControlPlaneSecurityGroup\"}],\"SubnetIds\":[{\"Ref\":\"SubnetPublicEUNORTH1B\"},{\"Ref\":\"SubnetPublicEUNORTH1A\"},{\"Ref\":\"SubnetPublicEUNORTH1C\"},{\"Ref\":\"SubnetPrivateEUNORTH1B\"},{\"Ref\":\"SubnetPrivateEUNORTH1A\"},{\"Ref\":\"SubnetPrivateEUNORTH1C\"}]},\"RoleArn\":{\"Fn::GetAtt\":\"ServiceRole.Arn\"},\"Version\":\"1.13\"}},\"ControlPlaneSecurityGroup\":{\"Type\":\"AWS::EC2::SecurityGroup\",\"Properties\":{\"GroupDescription\":\"Communication between the control plane and worker nodegroups\",\"Tags\":[{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/ControlPlaneSecurityGroup\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"IngressInterNodeGroupSG\":{\"Type\":\"AWS::EC2::SecurityGroupIngress\",\"Properties\":{\"Description\":\"Allow nodes to communicate with each other (all ports)\",\"FromPort\":0,\"GroupId\":{\"Ref\":\"ClusterSharedNodeSecurityGroup\"},\"IpProtocol\":\"-1\",\"SourceSecurityGroupId\":{\"Ref\":\"ClusterSharedNodeSecurityGroup\"},\"ToPort\":65535}},\"InternetGateway\":{\"Type\":\"AWS::EC2::InternetGateway\",\"Properties\":{\"Tags\":[{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/InternetGateway\"}}]}},\"NATGateway\":{\"Type\":\"AWS::EC2::NatGateway\",\"Properties\":{\"AllocationId\":{\"Fn::GetAtt\":\"NATIP.AllocationId\"},\"SubnetId\":{\"Ref\":\"SubnetPublicEUNORTH1C\"},\"Tags\":[{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/NATGateway\"}}]}},\"NATIP\":{\"Type\":\"AWS::EC2::EIP\",\"Properties\":{\"Domain\":\"vpc\"}},\"NATPrivateSubnetRouteEUNORTH1A\":{\"Type\":\"AWS::EC2::Route\",\"Properties\":{\"DestinationCidrBlock\":\"0.0.0.0/0\",\"NatGatewayId\":{\"Ref\":\"NATGateway\"},\"RouteTableId\":{\"Ref\":\"PrivateRouteTableEUNORTH1A\"}}},\"NATPrivateSubnetRouteEUNORTH1B\":{\"Type\":\"AWS::EC2::Route\",\"Properties\":{\"DestinationCidrBlock\":\"0.0.0.0/0\",\"NatGatewayId\":{\"Ref\":\"NATGateway\"},\"RouteTableId\":{\"Ref\":\"PrivateRouteTableEUNORTH1B\"}}},\"NATPrivateSubnetRouteEUNORTH1C\":{\"Type\":\"AWS::EC2::Route\",\"Properties\":{\"DestinationCidrBlock\":\"0.0.0.0/0\",\"NatGatewayId\":{\"Ref\":\"NATGateway\"},\"RouteTableId\":{\"Ref\":\"PrivateRouteTableEUNORTH1C\"}}},\"PolicyCloudWatchMetrics\":{\"Type\":\"AWS::IAM::Policy\",\"Properties\":{\"PolicyDocument\":{\"Statement\":[{\"Action\":[\"cloudwatch:PutMetricData\"],\"Effect\":\"Allow\",\"Resource\":\"*\"}],\"Version\":\"2012-10-17\"},\"PolicyName\":{\"Fn::Sub\":\"${AWS::StackName}-PolicyCloudWatchMetrics\"},\"Roles\":[{\"Ref\":\"ServiceRole\"}]}},\"PolicyNLB\":{\"Type\":\"AWS::IAM::Policy\",\"Properties\":{\"PolicyDocument\":{\"Statement\":[{\"Action\":[\"elasticloadbalancing:*\",\"ec2:CreateSecurityGroup\",\"ec2:Describe*\"],\"Effect\":\"Allow\",\"Resource\":\"*\"}],\"Version\":\"2012-10-17\"},\"PolicyName\":{\"Fn::Sub\":\"${AWS::StackName}-PolicyNLB\"},\"Roles\":[{\"Ref\":\"ServiceRole\"}]}},\"PrivateRouteTableEUNORTH1A\":{\"Type\":\"AWS::EC2::RouteTable\",\"Properties\":{\"Tags\":[{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/PrivateRouteTableEUNORTH1A\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"PrivateRouteTableEUNORTH1B\":{\"Type\":\"AWS::EC2::RouteTable\",\"Properties\":{\"Tags\":[{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/PrivateRouteTableEUNORTH1B\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"PrivateRouteTableEUNORTH1C\":{\"Type\":\"AWS::EC2::RouteTable\",\"Properties\":{\"Tags\":[{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/PrivateRouteTableEUNORTH1C\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"PublicRouteTable\":{\"Type\":\"AWS::EC2::RouteTable\",\"Properties\":{\"Tags\":[{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/PublicRouteTable\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"PublicSubnetRoute\":{\"Type\":\"AWS::EC2::Route\",\"Properties\":{\"DestinationCidrBlock\":\"0.0.0.0/0\",\"GatewayId\":{\"Ref\":\"InternetGateway\"},\"RouteTableId\":{\"Ref\":\"PublicRouteTable\"}}},\"RouteTableAssociationPrivateEUNORTH1A\":{\"Type\":\"AWS::EC2::SubnetRouteTableAssociation\",\"Properties\":{\"RouteTableId\":{\"Ref\":\"PrivateRouteTableEUNORTH1A\"},\"SubnetId\":{\"Ref\":\"SubnetPrivateEUNORTH1A\"}}},\"RouteTableAssociationPrivateEUNORTH1B\":{\"Type\":\"AWS::EC2::SubnetRouteTableAssociation\",\"Properties\":{\"RouteTableId\":{\"Ref\":\"PrivateRouteTableEUNORTH1B\"},\"SubnetId\":{\"Ref\":\"SubnetPrivateEUNORTH1B\"}}},\"RouteTableAssociationPrivateEUNORTH1C\":{\"Type\":\"AWS::EC2::SubnetRouteTableAssociation\",\"Properties\":{\"RouteTableId\":{\"Ref\":\"PrivateRouteTableEUNORTH1C\"},\"SubnetId\":{\"Ref\":\"SubnetPrivateEUNORTH1C\"}}},\"RouteTableAssociationPublicEUNORTH1A\":{\"Type\":\"AWS::EC2::SubnetRouteTableAssociation\",\"Properties\":{\"RouteTableId\":{\"Ref\":\"PublicRouteTable\"},\"SubnetId\":{\"Ref\":\"SubnetPublicEUNORTH1A\"}}},\"RouteTableAssociationPublicEUNORTH1B\":{\"Type\":\"AWS::EC2::SubnetRouteTableAssociation\",\"Properties\":{\"RouteTableId\":{\"Ref\":\"PublicRouteTable\"},\"SubnetId\":{\"Ref\":\"SubnetPublicEUNORTH1B\"}}},\"RouteTableAssociationPublicEUNORTH1C\":{\"Type\":\"AWS::EC2::SubnetRouteTableAssociation\",\"Properties\":{\"RouteTableId\":{\"Ref\":\"PublicRouteTable\"},\"SubnetId\":{\"Ref\":\"SubnetPublicEUNORTH1C\"}}},\"ServiceRole\":{\"Type\":\"AWS::IAM::Role\",\"Properties\":{\"AssumeRolePolicyDocument\":{\"Statement\":[{\"Action\":[\"sts:AssumeRole\"],\"Effect\":\"Allow\",\"Principal\":{\"Service\":[\"eks.amazonaws.com\"]}}],\"Version\":\"2012-10-17\"},\"ManagedPolicyArns\":[\"arn:aws:iam::aws:policy/AmazonEKSServicePolicy\",\"arn:aws:iam::aws:policy/AmazonEKSClusterPolicy\"]}},\"SubnetPrivateEUNORTH1A\":{\"Type\":\"AWS::EC2::Subnet\",\"Properties\":{\"AvailabilityZone\":\"eu-north-1a\",\"CidrBlock\":\"192.168.160.0/19\",\"Tags\":[{\"Key\":\"kubernetes.io/role/internal-elb\",\"Value\":\"1\"},{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/SubnetPrivateEUNORTH1A\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"SubnetPrivateEUNORTH1B\":{\"Type\":\"AWS::EC2::Subnet\",\"Properties\":{\"AvailabilityZone\":\"eu-north-1b\",\"CidrBlock\":\"192.168.128.0/19\",\"Tags\":[{\"Key\":\"kubernetes.io/role/internal-elb\",\"Value\":\"1\"},{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/SubnetPrivateEUNORTH1B\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"SubnetPrivateEUNORTH1C\":{\"Type\":\"AWS::EC2::Subnet\",\"Properties\":{\"AvailabilityZone\":\"eu-north-1c\",\"CidrBlock\":\"192.168.96.0/19\",\"Tags\":[{\"Key\":\"kubernetes.io/role/internal-elb\",\"Value\":\"1\"},{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/SubnetPrivateEUNORTH1C\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"SubnetPublicEUNORTH1A\":{\"Type\":\"AWS::EC2::Subnet\",\"Properties\":{\"AvailabilityZone\":\"eu-north-1a\",\"CidrBlock\":\"192.168.64.0/19\",\"Tags\":[{\"Key\":\"kubernetes.io/role/elb\",\"Value\":\"1\"},{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/SubnetPublicEUNORTH1A\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"SubnetPublicEUNORTH1B\":{\"Type\":\"AWS::EC2::Subnet\",\"Properties\":{\"AvailabilityZone\":\"eu-north-1b\",\"CidrBlock\":\"192.168.32.0/19\",\"Tags\":[{\"Key\":\"kubernetes.io/role/elb\",\"Value\":\"1\"},{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/SubnetPublicEUNORTH1B\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"SubnetPublicEUNORTH1C\":{\"Type\":\"AWS::EC2::Subnet\",\"Properties\":{\"AvailabilityZone\":\"eu-north-1c\",\"CidrBlock\":\"192.168.0.0/19\",\"Tags\":[{\"Key\":\"kubernetes.io/role/elb\",\"Value\":\"1\"},{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/SubnetPublicEUNORTH1C\"}}],\"VpcId\":{\"Ref\":\"VPC\"}}},\"VPC\":{\"Type\":\"AWS::EC2::VPC\",\"Properties\":{\"CidrBlock\":\"192.168.0.0/16\",\"EnableDnsHostnames\":true,\"EnableDnsSupport\":true,\"Tags\":[{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/VPC\"}}]}},\"VPCGatewayAttachment\":{\"Type\":\"AWS::EC2::VPCGatewayAttachment\",\"Properties\":{\"InternetGatewayId\":{\"Ref\":\"InternetGateway\"},\"VpcId\":{\"Ref\":\"VPC\"}}}},\"Outputs\":{\"ARN\":{\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}::ARN\"}},\"Value\":{\"Fn::GetAtt\":\"ControlPlane.Arn\"}},\"CertificateAuthorityData\":{\"Value\":{\"Fn::GetAtt\":\"ControlPlane.CertificateAuthorityData\"}},\"ClusterStackName\":{\"Value\":{\"Ref\":\"AWS::StackName\"}},\"Endpoint\":{\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}::Endpoint\"}},\"Value\":{\"Fn::GetAtt\":\"ControlPlane.Endpoint\"}},\"FeatureNATMode\":{\"Value\":\"Single\"},\"SecurityGroup\":{\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}::SecurityGroup\"}},\"Value\":{\"Ref\":\"ControlPlaneSecurityGroup\"}},\"ServiceRoleARN\":{\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}::ServiceRoleARN\"}},\"Value\":{\"Fn::GetAtt\":\"ServiceRole.Arn\"}},\"SharedNodeSecurityGroup\":{\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}::SharedNodeSecurityGroup\"}},\"Value\":{\"Ref\":\"ClusterSharedNodeSecurityGroup\"}},\"SubnetsPrivate\":{\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}::SubnetsPrivate\"}},\"Value\":{\"Fn::Join\":[\",\",[{\"Ref\":\"SubnetPrivateEUNORTH1B\"},{\"Ref\":\"SubnetPrivateEUNORTH1A\"},{\"Ref\":\"SubnetPrivateEUNORTH1C\"}]]}},\"SubnetsPublic\":{\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}::SubnetsPublic\"}},\"Value\":{\"Fn::Join\":[\",\",[{\"Ref\":\"SubnetPublicEUNORTH1B\"},{\"Ref\":\"SubnetPublicEUNORTH1A\"},{\"Ref\":\"SubnetPublicEUNORTH1C\"}]]}},\"VPC\":{\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}::VPC\"}},\"Value\":{\"Ref\":\"VPC\"}}}}"
}
2019-08-07T12:28:51+02:00 [ℹ]  deploying stack "eksctl-cluster-1-cluster"
2019-08-07T12:28:51+02:00 [▶]  start waiting for CloudFormation stack "eksctl-cluster-1-cluster" to reach "CREATE_COMPLETE" status
2019-08-07T12:28:51+02:00 [▶]  waiting for CloudFormation stack "eksctl-cluster-1-cluster" to reach "CREATE_COMPLETE" status

<repeated lines moved>

2019-08-07T12:39:51+02:00 [▶]  waiting for CloudFormation stack "eksctl-cluster-1-cluster" to reach "CREATE_COMPLETE" status
2019-08-07T12:39:51+02:00 [▶]  done after 10m59.887951392s of waiting for CloudFormation stack "eksctl-cluster-1-cluster" to reach "CREATE_COMPLETE" status
2019-08-07T12:39:51+02:00 [▶]  processing stack outputs
2019-08-07T12:39:51+02:00 [▶]  completed task: create cluster control plane "cluster-1"
2019-08-07T12:39:51+02:00 [▶]  started task: create nodegroup "ng-1"
2019-08-07T12:39:51+02:00 [▶]  waiting for 1 parallel tasks to complete
2019-08-07T12:39:51+02:00 [▶]  started task: create nodegroup "ng-1"
2019-08-07T12:39:51+02:00 [ℹ]  building nodegroup stack "eksctl-cluster-1-nodegroup-ng-1"
2019-08-07T12:39:51+02:00 [▶]  user-data = H4sIAAAAAAAA/6xa/XLiSJL/309R5+nYmA63BEICA7tcnBDCxkbYGIxt5uYchVRIZfQ1pRLC9PU9+0VJYCSVZse9sT0R41BWZlbmLz9B/GK6QWwJZuCvsX0WQnMDbRR1gR+77hmJfdOzumcCEEBtC0nNxataKlCLTIJDGtVCRATsRxT6JqqtgoBGlMBQhG5DjJyzhGCKXtfYRRHTYgY+RT7tgv89AwCAXwDaRCZ1hShEJl5jE0TvEUWeBSwShAL2QexjCtYBAZt4hVxEv6UPqgf3gQ/G2I93oAF+VceNr2epxt9miGyxiX4/6B8HJnSBhyi0IIUghAR6iCISdcGDfjW6m3wD6tPsdaAP1cfx/DWjpbK6v8Uk8D3k0yF2Ua+GqFnLzK0d9YnI3x4uunKDFXQB9C0QUUixWbhLGz/O5vrD62Ay+wYmdwP9daz29fHxYa6OJvPZX157wCB3a+aeH/hCxaWp6tH9NzCazObqRNNfR4NP3+EyzelNmcgOmTMKCe2VHmtxRGor7B8FwX+nDAAIgh9YSMBh78v3gyU/yocuXCE3OjJkkPz4Bt3QgWJmj4iD2jG7BGz1vnzPOZPX58GdEAYWU2aoz6/3d4NZ/pggG0cUkfTeHiUxyhMTTB2BQuzTD2OykORVQNcNEiEkeItdZCMr03I6z+ooJMEWW4j0YBLlDwOf6UdEILFPsYd6VmBuEMkjgmgSkI0QurGN/Z7p47y8j4UV9gULk14tCGnN9DGDvcTCqjjjYSFlPD6iopXjCgNLwP6awJxJ2IM26n35zipBv5296trDq6ppd4+T+Q/R2hARmUTMjouF8kOEaSXCJBLNwGM5VAthHCEBelZL6cqilLuaZUjWZriMy8jiO/TcImhVzCw9GecZAEHiI9IFJAhol/3vDIAQUqcLUplDMzn8/ZCNshYhWjWpLjC9Lk3bFbuOKUDEw1GEAz/qgvN6S1HOC51LSO3jweihWPADQh1B+uBgaB5rf6Iaes904zTjiiz6ZHB/N5rMew6lYdSt1dROvV1vaLrW11RFaQ/kwXDYaWj1oSp1+u3BUHx3FfF0HyuWYiiK6k/x7LXqDaUuSYqsNBt/CWB1u/scQLmS7pUr+oiCDz10goQrfFarNgniMGP07QNsuQLtfdaHYu/8KxfSa6BlERRFXVAX0/8yYogXiDDJ7nEmiYfs3bQjZvVWWiEKM0thTB3kU2xCyiQOmQ39wH/3gjg6EgBAPly5yOqCNXQjdCAnaOUEwebEZULTQfP5uAsaXj3ihVlHOlB3zXonJ+hi5FNNZe2+iIsJRZPQD2MDgvcFW73AQl3wlFnyT+1SD+LISi1s5i1MGR59WGSRDxxmGuQBwVsWx+xpfTjKkmMwmWW3CUCqi1KdxUOqFzgCD2K/e3zM5lfKsEaQxgRdQYo+AH8IKKToNosfWxkQ0RChbAWBFOWA3GDf6oIDo5bGOSYpQOlxlIrOx7P+cfH5kP2prDy0s0+lpcD+9fWr0QRo+sN8NBxp6lxPqem5MRpp7wNNU1earSajvmqP+upAnfTtzR/OBl91knpfnUZDdaAujAcj0acvg8V0OtCTkFhPNxloptd0ravF3ugrz4O53jQGdjLZ64mxf6nfDYPnwXyU0QYftGQ41+dGX79SpUc96z874xbKC7x8vomXzw+uudc0dTZKBtOXm9tgOXK25kSd6v3+VB3Ytn6vMqungWbbel8dHyDuo9r7BD8ai7tde/sH7bw97DoPN3hgjZD7OL6IhrY7nrn3Ed22Deu9XjNXziZ8Wj22oLa7cobzVMmFCZMdHU4whcuL4M71Z8ht9a/Vl9nd03082z3uHVlZOO79xUJWlpYV3mjhvnPfaCuSs7x5v8+AhU5neesNLIiurb32x2QSvz1JQ+1a9Z9R8tBqWbY1Xow6y068bbfNxFtMk/5Ewn1S2xvvb/tUyY2TvC+N0VZ6m271yYhS3HgJw6eFPWrXG2/m236xUjf46c1a3Q0Mb992nJsJrQ1irxFZ5gZmSyJy8MP++mJmJqtZ6yW5qHWmLQV6D/jdHSbesnVZvxi/uc/L62AvP6PNaH29S/pay9i1aRw2zE6GycvlS2v4sGvtV1ZfblqRZmhqoqsqNN4MTU8G9stg8VC/V6fXtb46Haj27cZQ2yy8lp7o/VRJLZkODdXoq+t2OazjQ1j76vWo8WisRqFy+dRu3/i3yv3Ma/Un6/0Up0relVr0cKtLQV/p36BADpuJ/fyiJNrdtNO+I8Hb7ROuyy/S4up52t9dXjYU3Y3kd+3xzWoY7VrWalvGJcZXQXO8eto1g8HF/XzbvDFgIMuhPLad20u9HkbJjDRb037N199NR2snl6voyVms68tNVu57NDCMaNlw1sZsMbrdvry58uShFU4jIm0N61IJmomzXDuTF+vybUvM0XZpNN0keakvdiP1TUuVEDyjN1fuXLE8KnnSk/nmO/Gz7A9a2o0zlJ+N5W3rzYumUhtamu7H/Zb+h9peRHfj/hKpt0YG7NXMbulL+0HDHTyhl3dYbtrm87ttmdHleG7c34/m0vK6pT8vhnfX6svVrFnfJVtPuw16pzahTwZVTeJTveljMnxuWObm4lbKd+bo2LkPz7mxcWq5wmE00Pc/HU+nftsF/949CQC2X3zMDsZXEMnWkcwn5vWOnnzKnvOTNnPyM8rYvzhizIc58F+fEsqM/QkRMyYE+VQ4GvszstkAzCZfSggJWiOCfBNFXfD9x9nRiQ9Iftq8AwindWaHzNNTMbWyRUYs7lWnzSvdIKW8LLGjvC4B0GCD/AJFwIXH4op+iGrgeZABAZNIwNATcgYEJMeI/O3hq5Of2AByn34+W260IUYedF0gZWZSGYoeNkkAlPTZkw7n7exREV1IbAQa2dLkNcVdRmhmDEQRGyVK0yrzNI9KsgFiyqJy4GjIypHULpOaZc12o0zxmtASG0dll/LlwSexgo2zSWqUbiRNyCsjTJZjpY0CagzFon5TKfnc9EscWEY+548iSvXyVbLoQ/94E+aB8uTiVcyNMqHsvtm0ynczC6V2yXvMhyq1m3O2rKtoAS6Z6JUjRHh70nta5RDJZbadhColuZB5zXKMCO8clKpSk8eOA8rm4+JIfGlUpSvzQCo7yoi7AmZ7yRLlElfIwUE4a5WqyuLsX/OOe7LoIQvHHpAauWorZaxUiiyrIa76eFCbViWtoMvi6n0t5ZBSGsfcKjKx2jzYfQJYKfIwMCvyv0SyGnyxSVxvqQhp2OAjSrj8M5sVGIgdjuSXIebzqqoJHruA3Dx1+gIupsLHnIW4TCS8naQ8B2gj36VIRaZQuXS9zaOb9cSyK165vXil9kKqHKno2l45w5hNhTbOD8cCwasAmqU3P0q4nlyRJaRsDsvKD1VHLljOOCdXBNmXcZlnXIz4lkTKSLIby4mKuabinQpRkk6TqizocbuAxwfG5LTbMu+Pzbdmr3IQe9y2wVziUC2lolQM+8k95SMRCrXj8N0RV2VvRWAa3KSDJft2Eg+lWa4v1hzLncGWI37RqRj9ZdPph03KcdiIchlYJ5J4B7mGHMpWRcGSig7FmwGlY0P4aO5Vg1wu3cgByg84wg8h2ihGFErcrOEXHdFDFLqnSpf5MuBbI2sapcsh5xlr6hUZxblaUcDcrlexx1YUChuCXJBNubylcosi4dPA47EK5Yp2wO90zIh26QKptGUwaMqZbpWhcSqs4grLK6v2Kpb+sLJLcnXMoppfw6vSBXGDuuLzhFlu5ybXNU2zIvAVSxQttdLPvZOBu9cwsCLRg+FnPzT+8h/pm9oVjJzsrW6EKBACgAhBO0zzpBCHaA2xm6f5QexHiGai69g32WdvYCP6ejTm16/ge3oKQOJgFwGCoAWO729f6XuIAOP7O7BO34DgNQC//QbOv3wvMP44B70eo0o/zsHvv4O//e3AxRSww/8D//NbXej8fvGFHf8dsA/kH0oBQKYTgAN3jkwQjcmJb3387G8FPgL/+GcQA/Ajc/3wNrt3/uVXMyYuEIQIu8in6bdS3VpNanXERpMlX/q35kKKIpq+RRMsSGEtfTch4HCrfD1PNebeaf/rWnOvyctq5y/3+r9BMQvL1/ND6gQxMdGfvWIDvwDq4AiY0AfBFhGCLQSO7+UzBSak4D8r5T9+egD+8Q/9bljA/PRbAg644k8DKgDIMbDnjOVoVO53A13hy6/5rGZJV5Q8/5oJM+syNNI3zCZ1gQWRF/gCQW4ArdJZ9p7u+NVY6TCikNDc2Z93gX/ldzhch7hsNs/P/j8AAP//o1CKsvwjAAA=
2019-08-07T12:39:51+02:00 [ℹ]  --nodes-min=1 was set automatically for nodegroup ng-1
2019-08-07T12:39:51+02:00 [ℹ]  --nodes-max=1 was set automatically for nodegroup ng-1
2019-08-07T12:39:51+02:00 [▶]  CreateStackInput = {
  Capabilities: ["CAPABILITY_IAM"],
  StackName: "eksctl-cluster-1-nodegroup-ng-1",
  Tags: [
    {
      Key: "alpha.eksctl.io/cluster-name",
      Value: "cluster-1"
    },
    {
      Key: "eksctl.cluster.k8s.io/v1alpha1/cluster-name",
      Value: "cluster-1"
    },
    {
      Key: "eksctl.io/v1alpha2/nodegroup-name",
      Value: "ng-1"
    },
    {
      Key: "alpha.eksctl.io/nodegroup-name",
      Value: "ng-1"
    }
  ],
  TemplateBody: "{\"AWSTemplateFormatVersion\":\"2010-09-09\",\"Description\":\"EKS nodes (AMI family: AmazonLinux2, SSH access: false, private networking: false) [created and managed by eksctl]\",\"Resources\":{\"EgressInterCluster\":{\"Type\":\"AWS::EC2::SecurityGroupEgress\",\"Properties\":{\"Description\":\"Allow control plane to communicate with worker nodes in group ng-1 (kubelet and workload TCP ports)\",\"DestinationSecurityGroupId\":{\"Ref\":\"SG\"},\"FromPort\":1025,\"GroupId\":{\"Fn::ImportValue\":\"eksctl-cluster-1-cluster::SecurityGroup\"},\"IpProtocol\":\"tcp\",\"ToPort\":65535}},\"EgressInterClusterAPI\":{\"Type\":\"AWS::EC2::SecurityGroupEgress\",\"Properties\":{\"Description\":\"Allow control plane to communicate with worker nodes in group ng-1 (workloads using HTTPS port, commonly used with extension API servers)\",\"DestinationSecurityGroupId\":{\"Ref\":\"SG\"},\"FromPort\":443,\"GroupId\":{\"Fn::ImportValue\":\"eksctl-cluster-1-cluster::SecurityGroup\"},\"IpProtocol\":\"tcp\",\"ToPort\":443}},\"IngressInterCluster\":{\"Type\":\"AWS::EC2::SecurityGroupIngress\",\"Properties\":{\"Description\":\"Allow worker nodes in group ng-1 to communicate with control plane (kubelet and workload TCP ports)\",\"FromPort\":1025,\"GroupId\":{\"Ref\":\"SG\"},\"IpProtocol\":\"tcp\",\"SourceSecurityGroupId\":{\"Fn::ImportValue\":\"eksctl-cluster-1-cluster::SecurityGroup\"},\"ToPort\":65535}},\"IngressInterClusterAPI\":{\"Type\":\"AWS::EC2::SecurityGroupIngress\",\"Properties\":{\"Description\":\"Allow worker nodes in group ng-1 to communicate with control plane (workloads using HTTPS port, commonly used with extension API servers)\",\"FromPort\":443,\"GroupId\":{\"Ref\":\"SG\"},\"IpProtocol\":\"tcp\",\"SourceSecurityGroupId\":{\"Fn::ImportValue\":\"eksctl-cluster-1-cluster::SecurityGroup\"},\"ToPort\":443}},\"IngressInterClusterCP\":{\"Type\":\"AWS::EC2::SecurityGroupIngress\",\"Properties\":{\"Description\":\"Allow control plane to receive API requests from worker nodes in group ng-1\",\"FromPort\":443,\"GroupId\":{\"Fn::ImportValue\":\"eksctl-cluster-1-cluster::SecurityGroup\"},\"IpProtocol\":\"tcp\",\"SourceSecurityGroupId\":{\"Ref\":\"SG\"},\"ToPort\":443}},\"NodeGroup\":{\"Type\":\"AWS::AutoScaling::AutoScalingGroup\",\"Properties\":{\"DesiredCapacity\":\"1\",\"LaunchTemplate\":{\"LaunchTemplateName\":{\"Fn::Sub\":\"${AWS::StackName}\"},\"Version\":{\"Fn::GetAtt\":\"NodeGroupLaunchTemplate.LatestVersionNumber\"}},\"MaxSize\":\"1\",\"MinSize\":\"1\",\"Tags\":[{\"Key\":\"Name\",\"PropagateAtLaunch\":\"true\",\"Value\":\"cluster-1-ng-1-Node\"},{\"Key\":\"kubernetes.io/cluster/cluster-1\",\"PropagateAtLaunch\":\"true\",\"Value\":\"owned\"}],\"VPCZoneIdentifier\":{\"Fn::Split\":[\",\",{\"Fn::ImportValue\":\"eksctl-cluster-1-cluster::SubnetsPublic\"}]}},\"UpdatePolicy\":{\"AutoScalingRollingUpdate\":{\"MaxBatchSize\":\"1\",\"MinInstancesInService\":\"0\"}}},\"NodeGroupLaunchTemplate\":{\"Type\":\"AWS::EC2::LaunchTemplate\",\"Properties\":{\"LaunchTemplateData\":{\"IamInstanceProfile\":{\"Arn\":{\"Fn::GetAtt\":\"NodeInstanceProfile.Arn\"}},\"ImageId\":\"ami-0fd05922165907b85\",\"InstanceType\":\"m5.large\",\"NetworkInterfaces\":[{\"AssociatePublicIpAddress\":true,\"DeviceIndex\":0,\"Groups\":[{\"Fn::ImportValue\":\"eksctl-cluster-1-cluster::SharedNodeSecurityGroup\"},{\"Ref\":\"SG\"}]}],\"UserData\":\"H4sIAAAAAAAA/6xa/XLiSJL/309R5+nYmA63BEICA7tcnBDCxkbYGIxt5uYchVRIZfQ1pRLC9PU9+0VJYCSVZse9sT0R41BWZlbmLz9B/GK6QWwJZuCvsX0WQnMDbRR1gR+77hmJfdOzumcCEEBtC0nNxataKlCLTIJDGtVCRATsRxT6JqqtgoBGlMBQhG5DjJyzhGCKXtfYRRHTYgY+RT7tgv89AwCAXwDaRCZ1hShEJl5jE0TvEUWeBSwShAL2QexjCtYBAZt4hVxEv6UPqgf3gQ/G2I93oAF+VceNr2epxt9miGyxiX4/6B8HJnSBhyi0IIUghAR6iCISdcGDfjW6m3wD6tPsdaAP1cfx/DWjpbK6v8Uk8D3k0yF2Ua+GqFnLzK0d9YnI3x4uunKDFXQB9C0QUUixWbhLGz/O5vrD62Ay+wYmdwP9daz29fHxYa6OJvPZX157wCB3a+aeH/hCxaWp6tH9NzCazObqRNNfR4NP3+EyzelNmcgOmTMKCe2VHmtxRGor7B8FwX+nDAAIgh9YSMBh78v3gyU/yocuXCE3OjJkkPz4Bt3QgWJmj4iD2jG7BGz1vnzPOZPX58GdEAYWU2aoz6/3d4NZ/pggG0cUkfTeHiUxyhMTTB2BQuzTD2OykORVQNcNEiEkeItdZCMr03I6z+ooJMEWW4j0YBLlDwOf6UdEILFPsYd6VmBuEMkjgmgSkI0QurGN/Z7p47y8j4UV9gULk14tCGnN9DGDvcTCqjjjYSFlPD6iopXjCgNLwP6awJxJ2IM26n35zipBv5296trDq6ppd4+T+Q/R2hARmUTMjouF8kOEaSXCJBLNwGM5VAthHCEBelZL6cqilLuaZUjWZriMy8jiO/TcImhVzCw9GecZAEHiI9IFJAhol/3vDIAQUqcLUplDMzn8/ZCNshYhWjWpLjC9Lk3bFbuOKUDEw1GEAz/qgvN6S1HOC51LSO3jweihWPADQh1B+uBgaB5rf6Iaes904zTjiiz6ZHB/N5rMew6lYdSt1dROvV1vaLrW11RFaQ/kwXDYaWj1oSp1+u3BUHx3FfF0HyuWYiiK6k/x7LXqDaUuSYqsNBt/CWB1u/scQLmS7pUr+oiCDz10goQrfFarNgniMGP07QNsuQLtfdaHYu/8KxfSa6BlERRFXVAX0/8yYogXiDDJ7nEmiYfs3bQjZvVWWiEKM0thTB3kU2xCyiQOmQ39wH/3gjg6EgBAPly5yOqCNXQjdCAnaOUEwebEZULTQfP5uAsaXj3ihVlHOlB3zXonJ+hi5FNNZe2+iIsJRZPQD2MDgvcFW73AQl3wlFnyT+1SD+LISi1s5i1MGR59WGSRDxxmGuQBwVsWx+xpfTjKkmMwmWW3CUCqi1KdxUOqFzgCD2K/e3zM5lfKsEaQxgRdQYo+AH8IKKToNosfWxkQ0RChbAWBFOWA3GDf6oIDo5bGOSYpQOlxlIrOx7P+cfH5kP2prDy0s0+lpcD+9fWr0QRo+sN8NBxp6lxPqem5MRpp7wNNU1earSajvmqP+upAnfTtzR/OBl91knpfnUZDdaAujAcj0acvg8V0OtCTkFhPNxloptd0ravF3ugrz4O53jQGdjLZ64mxf6nfDYPnwXyU0QYftGQ41+dGX79SpUc96z874xbKC7x8vomXzw+uudc0dTZKBtOXm9tgOXK25kSd6v3+VB3Ytn6vMqungWbbel8dHyDuo9r7BD8ai7tde/sH7bw97DoPN3hgjZD7OL6IhrY7nrn3Ed22Deu9XjNXziZ8Wj22oLa7cobzVMmFCZMdHU4whcuL4M71Z8ht9a/Vl9nd03082z3uHVlZOO79xUJWlpYV3mjhvnPfaCuSs7x5v8+AhU5neesNLIiurb32x2QSvz1JQ+1a9Z9R8tBqWbY1Xow6y068bbfNxFtMk/5Ewn1S2xvvb/tUyY2TvC+N0VZ6m271yYhS3HgJw6eFPWrXG2/m236xUjf46c1a3Q0Mb992nJsJrQ1irxFZ5gZmSyJy8MP++mJmJqtZ6yW5qHWmLQV6D/jdHSbesnVZvxi/uc/L62AvP6PNaH29S/pay9i1aRw2zE6GycvlS2v4sGvtV1ZfblqRZmhqoqsqNN4MTU8G9stg8VC/V6fXtb46Haj27cZQ2yy8lp7o/VRJLZkODdXoq+t2OazjQ1j76vWo8WisRqFy+dRu3/i3yv3Ma/Un6/0Up0relVr0cKtLQV/p36BADpuJ/fyiJNrdtNO+I8Hb7ROuyy/S4up52t9dXjYU3Y3kd+3xzWoY7VrWalvGJcZXQXO8eto1g8HF/XzbvDFgIMuhPLad20u9HkbJjDRb037N199NR2snl6voyVms68tNVu57NDCMaNlw1sZsMbrdvry58uShFU4jIm0N61IJmomzXDuTF+vybUvM0XZpNN0keakvdiP1TUuVEDyjN1fuXLE8KnnSk/nmO/Gz7A9a2o0zlJ+N5W3rzYumUhtamu7H/Zb+h9peRHfj/hKpt0YG7NXMbulL+0HDHTyhl3dYbtrm87ttmdHleG7c34/m0vK6pT8vhnfX6svVrFnfJVtPuw16pzahTwZVTeJTveljMnxuWObm4lbKd+bo2LkPz7mxcWq5wmE00Pc/HU+nftsF/949CQC2X3zMDsZXEMnWkcwn5vWOnnzKnvOTNnPyM8rYvzhizIc58F+fEsqM/QkRMyYE+VQ4GvszstkAzCZfSggJWiOCfBNFXfD9x9nRiQ9Iftq8AwindWaHzNNTMbWyRUYs7lWnzSvdIKW8LLGjvC4B0GCD/AJFwIXH4op+iGrgeZABAZNIwNATcgYEJMeI/O3hq5Of2AByn34+W260IUYedF0gZWZSGYoeNkkAlPTZkw7n7exREV1IbAQa2dLkNcVdRmhmDEQRGyVK0yrzNI9KsgFiyqJy4GjIypHULpOaZc12o0zxmtASG0dll/LlwSexgo2zSWqUbiRNyCsjTJZjpY0CagzFon5TKfnc9EscWEY+548iSvXyVbLoQ/94E+aB8uTiVcyNMqHsvtm0ynczC6V2yXvMhyq1m3O2rKtoAS6Z6JUjRHh70nta5RDJZbadhColuZB5zXKMCO8clKpSk8eOA8rm4+JIfGlUpSvzQCo7yoi7AmZ7yRLlElfIwUE4a5WqyuLsX/OOe7LoIQvHHpAauWorZaxUiiyrIa76eFCbViWtoMvi6n0t5ZBSGsfcKjKx2jzYfQJYKfIwMCvyv0SyGnyxSVxvqQhp2OAjSrj8M5sVGIgdjuSXIebzqqoJHruA3Dx1+gIupsLHnIW4TCS8naQ8B2gj36VIRaZQuXS9zaOb9cSyK165vXil9kKqHKno2l45w5hNhTbOD8cCwasAmqU3P0q4nlyRJaRsDsvKD1VHLljOOCdXBNmXcZlnXIz4lkTKSLIby4mKuabinQpRkk6TqizocbuAxwfG5LTbMu+Pzbdmr3IQe9y2wVziUC2lolQM+8k95SMRCrXj8N0RV2VvRWAa3KSDJft2Eg+lWa4v1hzLncGWI37RqRj9ZdPph03KcdiIchlYJ5J4B7mGHMpWRcGSig7FmwGlY0P4aO5Vg1wu3cgByg84wg8h2ihGFErcrOEXHdFDFLqnSpf5MuBbI2sapcsh5xlr6hUZxblaUcDcrlexx1YUChuCXJBNubylcosi4dPA47EK5Yp2wO90zIh26QKptGUwaMqZbpWhcSqs4grLK6v2Kpb+sLJLcnXMoppfw6vSBXGDuuLzhFlu5ybXNU2zIvAVSxQttdLPvZOBu9cwsCLRg+FnPzT+8h/pm9oVjJzsrW6EKBACgAhBO0zzpBCHaA2xm6f5QexHiGai69g32WdvYCP6ejTm16/ge3oKQOJgFwGCoAWO729f6XuIAOP7O7BO34DgNQC//QbOv3wvMP44B70eo0o/zsHvv4O//e3AxRSww/8D//NbXej8fvGFHf8dsA/kH0oBQKYTgAN3jkwQjcmJb3387G8FPgL/+GcQA/Ajc/3wNrt3/uVXMyYuEIQIu8in6bdS3VpNanXERpMlX/q35kKKIpq+RRMsSGEtfTch4HCrfD1PNebeaf/rWnOvyctq5y/3+r9BMQvL1/ND6gQxMdGfvWIDvwDq4AiY0AfBFhGCLQSO7+UzBSak4D8r5T9+egD+8Q/9bljA/PRbAg644k8DKgDIMbDnjOVoVO53A13hy6/5rGZJV5Q8/5oJM+syNNI3zCZ1gQWRF/gCQW4ArdJZ9p7u+NVY6TCikNDc2Z93gX/ldzhch7hsNs/P/j8AAP//o1CKsvwjAAA=\"},\"LaunchTemplateName\":{\"Fn::Sub\":\"${AWS::StackName}\"}}},\"NodeInstanceProfile\":{\"Type\":\"AWS::IAM::InstanceProfile\",\"Properties\":{\"Path\":\"/\",\"Roles\":[{\"Ref\":\"NodeInstanceRole\"}]}},\"NodeInstanceRole\":{\"Type\":\"AWS::IAM::Role\",\"Properties\":{\"AssumeRolePolicyDocument\":{\"Statement\":[{\"Action\":[\"sts:AssumeRole\"],\"Effect\":\"Allow\",\"Principal\":{\"Service\":[\"ec2.amazonaws.com\"]}}],\"Version\":\"2012-10-17\"},\"ManagedPolicyArns\":[\"arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy\",\"arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy\",\"arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly\"],\"Path\":\"/\"}},\"SG\":{\"Type\":\"AWS::EC2::SecurityGroup\",\"Properties\":{\"GroupDescription\":\"Communication between the control plane and worker nodes in group ng-1\",\"Tags\":[{\"Key\":\"kubernetes.io/cluster/cluster-1\",\"Value\":\"owned\"},{\"Key\":\"Name\",\"Value\":{\"Fn::Sub\":\"${AWS::StackName}/SG\"}}],\"VpcId\":{\"Fn::ImportValue\":\"eksctl-cluster-1-cluster::VPC\"}}}},\"Outputs\":{\"FeatureLocalSecurityGroup\":{\"Value\":true},\"FeaturePrivateNetworking\":{\"Value\":false},\"FeatureSharedSecurityGroup\":{\"Value\":true},\"InstanceProfileARN\":{\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}::InstanceProfileARN\"}},\"Value\":{\"Fn::GetAtt\":\"NodeInstanceProfile.Arn\"}},\"InstanceRoleARN\":{\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}::InstanceRoleARN\"}},\"Value\":{\"Fn::GetAtt\":\"NodeInstanceRole.Arn\"}}}}"
}
2019-08-07T12:39:51+02:00 [ℹ]  deploying stack "eksctl-cluster-1-nodegroup-ng-1"
2019-08-07T12:39:51+02:00 [▶]  start waiting for CloudFormation stack "eksctl-cluster-1-nodegroup-ng-1" to reach "CREATE_COMPLETE" status
2019-08-07T12:39:51+02:00 [▶]  waiting for CloudFormation stack "eksctl-cluster-1-nodegroup-ng-1" to reach "CREATE_COMPLETE" status

<repeated lines removed>

2019-08-07T12:43:38+02:00 [▶]  waiting for CloudFormation stack "eksctl-cluster-1-nodegroup-ng-1" to reach "CREATE_COMPLETE" status
2019-08-07T12:43:38+02:00 [▶]  done after 3m46.674833102s of waiting for CloudFormation stack "eksctl-cluster-1-nodegroup-ng-1" to reach "CREATE_COMPLETE" status
2019-08-07T12:43:38+02:00 [▶]  processing stack outputs
2019-08-07T12:43:38+02:00 [▶]  completed task: create nodegroup "ng-1"
2019-08-07T12:43:38+02:00 [▶]  completed task: create nodegroup "ng-1"
2019-08-07T12:43:38+02:00 [▶]  started task: 1 task: { update CloudWatch logging configuration }
2019-08-07T12:43:38+02:00 [▶]  started task: update CloudWatch logging configuration
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x23172de]

goroutine 483 [running]:
github.com/weaveworks/eksctl/pkg/eks.(*updateClusterConfigTask).Do(0xc0003b0030, 0xc000049500, 0xc000049500, 0x1)
	/src/pkg/eks/update.go:36 +0x2e
github.com/weaveworks/eksctl/pkg/cfn/manager.doSingleTask(0xc000049380, 0x3c85900, 0xc0003b0030, 0x215fb2a)
	/src/pkg/cfn/manager/tasks.go:198 +0xec
github.com/weaveworks/eksctl/pkg/cfn/manager.doSequentialTasks(0xc000049380, 0xc0006c8710, 0x1, 0x1)
	/src/pkg/cfn/manager/tasks.go:228 +0x70
created by github.com/weaveworks/eksctl/pkg/cfn/manager.(*TaskTree).Do
	/src/pkg/cfn/manager/tasks.go:89 +0x159
```

--- a/pkg/eks/update.go
+++ b/pkg/eks/update.go
@@ -20,18 +20,12 @@ import (
 
 type updateClusterConfigTask struct {
 	info string
-	skip bool
 	spec *api.ClusterConfig
 	call func(*api.ClusterConfig) error
 }
 
-func (t *updateClusterConfigTask) Skip() bool { return t.skip }
-func (t *updateClusterConfigTask) Describe() string {
-	if t.skip {
-		return "(skip) " + t.info
-	}
-	return t.info
-}
+func (t *updateClusterConfigTask) Describe() string { return t.info }
+
 func (t *updateClusterConfigTask) Do(errs chan error) error {
 	err := t.call(t.spec)
 	close(errs)
@@ -126,6 +120,8 @@ func (c *ClusterProvider) GetUpdateClusterConfigTasks(cfg *api.ClusterConfig) *m
 	loggingTasks := &manager.TaskTree{Parallel: false}
 	loggingTasks.Append(&updateClusterConfigTask{
 		info: "update CloudWatch logging configuration",
+		spec: cfg,
+		call: c.UpdateClusterConfigForLogging,
 	})
 	return loggingTasks
 }

